.eslintrc.js

```diff
@@ -1,12 +1,15 @@
 module.exports = {
     "env": {
         "browser": true,
-        "es6": true
+        "es6": true,
+        "jest": true
```

Comment:
Deffo missing a rule for commas at the end of lists. Worth finding for this branch.

Reply:
Are you saying commas at the end of lists should be enforced? We found a prettier rule but it enforces that there *aren't commas*. Pls could u clarify? Thx